src/stories/components/CioPlpGrid/RenderProps.md

```diff
@@ -1,5 +1,18 @@
-| property | type                               | description                        |
-| -------- | ---------------------------------- | ---------------------------------- |
-| data     | `Nullable<ConstructorIOClient>`    | The Constructor IO client instance |
-| status   | `CioClientOptions`                 | The Constructor IO client options  |
-| refetch  | `React.Dispatch<CioClientOptions>` | The Constructor IO options setter  |
+| property   | type                                                                                                                                 | description                             |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------- | ------- | ------ | ----------------- |
+| status     | `stale                                                                                                                               | fetching                                | success | error` | API request state |
+| data       | [PlpSearchData](./?path=/docs/hooks-usesearchresults--props) \| [PlpBrowseData](./?path=/docs/hooks-usebrowseresults--props) \| null | API response data                       |
+| refetch    | `() => void`                                                                                                                         | Function that refetches the search data |
+| sort       | [Sort Options](./?path=/docs/components-sort--code-examples#arguments-passed-to-children-via-render-props)                           | Sort object                             |
+| filters    | [Filter Options](./?path=/docs/components-filters--code-examples#arguments-passed-to-children-via-render-props)                      | Filter object                           |
+| groups     | [Group Options](./?path=/docs/components-groups--code-examples#arguments-passed-to-children-via-render-props)                        | Group object                            |
+| pagination | [Pagination Options](./?path=/docs/components-pagination--code-examples#arguments-passed-to-children-via-render-props)               | Pagination object                       |
```

Comment:
Should we also include:
- isSearchPage
- isBrowsePage
- searchQuery
- browseFilterName
- browseFilterValue

from the return of [useCioPlp](https://github.com/Constructor-io/constructorio-ui-plp/blob/main/src/hooks/useCioPlp.ts)